Automattic/wp-calypso
# Improve ICANN verification flow

Fixes #2478 #13231 
Also as a bonus, added a fix for #6098

This improves the presentation of the ICANN email verification step.

Before | After
------- | -------
![image](https://user-images.githubusercontent.com/12596797/28026935-81c93576-6565-11e7-8f15-f66ef2a916b6.png) | ![image](https://user-images.githubusercontent.com/12596797/28090922-12ed912c-665c-11e7-8225-64393c74384d.png)
![image](https://user-images.githubusercontent.com/12596797/28026990-ae0cf00a-6565-11e7-8c42-35919a6c271d.png) | ![image](https://user-images.githubusercontent.com/12596797/28090954-291c4916-665c-11e7-8a26-d2a459b3afc2.png)




Changes:
- Updated copy for ICANN verification card - removed all mention of ICANN.
- ^^ That includes adding specific mention of what email address the email was and will be sent to and a direct link to change it.
- Replace global success notices with a status bar that communicates the action needed.


How to test:
- Purchase a domain but don't verify your email address yet.
- Go to Domains and click on the domain you just bought.
- Affirm you see the Email verification box at the top.
- Click "Send Again".
- Affirm you were shown a "Email sent" message.
- After 5 seconds the message should disappear. Then click "Change Email Address".
- Affirm you are taken to the Edit Contact Info screen.
